Character/Action_Dash.gd

```diff
@@ -0,0 +1,16 @@
+extends Node
```

Comment:
delete this whole file? Why is there GD code here x)

Reply:
Really good point, don't even know where that came from >_> consider it gone